File: data structures and algorithms/java/06_064.java
// Creating a trie and inserting words
Trie trie = new Trie();
trie.insert("apple");
trie.insert("app");
trie.insert("apricot");
System.out.println(trie.search("apple"));    // true
System.out.println(trie.search("app"));      // true
System.out.println(trie.search("appl"));     // false
System.out.println(trie.startsWith("app"));  // true

File: data structures and algorithms/cs/06_064.cs
// Creating a trie and inserting words
Trie trie = new Trie();
trie.Insert("apple");
trie.Insert("app");
trie.Insert("apricot");
Console.WriteLine(trie.Search("apple"));    // true
Console.WriteLine(trie.Search("app"));      // true
Console.WriteLine(trie.Search("appl"));     // false
Console.WriteLine(trie.StartsWith("app"));  // true